app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/model/DatabaseService.kt

```diff
@@ -184,6 +184,12 @@ interface DatabaseService {
 
     suspend fun updateUser(firebaseUser: FirebaseUser): User
 
+    suspend fun setAdminAccess(firebaseUser: FirebaseUser, adminAccess: Boolean): User
+
+    suspend fun checkIsAdmin(user : User): Boolean
```

Comment:
is this method redundant? should it refresh into the database?

Reply:
Yes it should, should directly implement this in UpdateUser with argument AdminAccess ?

Reply:
I can see that all the implementations right now just check the boolean, but if this is more futureproof go ahead.

>should [I] directly implement this in UpdateUser with argument AdminAccess ?
`setAdminAccess` is fine, I was asking about `checkIsAdmin`

Reply:
It is not clear at all from the perspective of the caller what the difference between `user.isAdmin` and `checkIsAdmin(user)` is. Please explain